src/dbsync/tests/string_helper/string_helper_test.h

```diff
@@ -0,0 +1,12 @@
+#include "gtest/gtest.h"
+
+class StringHelperTest : public ::testing::Test {
+
+protected:
+
+    StringHelperTest() = default;
+    virtual ~StringHelperTest() = default;
+
+    virtual void SetUp();
```

Comment:
Please, change virtual here to use the _override_ specifier in both SetUp and TearDown methods.